Src/AutoFixture.xUnit.net2.UnitTest/Scenario.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using System.ComponentModel.DataAnnotations;
```

Comment:
According to this [MSDN page](https://msdn.microsoft.com/en-us/library/system.componentmodel.dataannotations.aspx?f=255&MSPPError=-2147217396) and [source code](https://github.com/dotnet/corefx/tree/master/src/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations) I assume it does.